server/routes/referrals/referralsController.test.ts

```diff
@@ -236,7 +236,7 @@ describe('POST /referrals/:id/complexity-level', () => {
       .type('form')
       .send({ 'complexity-level-id': 'd0db50b0-4a50-4fc7-a006-9c97530e38b2' })
       .expect(302)
-      .expect('Location', '/referrals/1/form')
+      .expect('Location', '/referrals/1/completion-deadline')
```

Comment:
It's a bit of a pain, but do we want to update the `it` for these too? Still says "redirects to the referral form".

Reply:
🤦‍♂️ rush job, good catch. Will change!